### ISSUE TITLE
Warning glow for Reaping mobs

### ISSUE THREAD

**Issue by Nukme** [Luxocracy/NeatPlates#76]
version: 6.22.8-beta2
theme: neon
role: tank

I noticed that you've implemented a new color-change method for Reaping mobs. I tested it in M+ to check a flaw and it's still there.

If a tank player enable Waning Glow, it would show red spikes at left/right end of the health bar if the mobs are not aggro-ed on the tank. The work-around for reaping mobs does fix the color-change, but since the threat system is broken, the warning glows are always there, which would result in blue(safe) health bars with red spikes. 

Hope you could look into this and make warning glows consistent with normal threat behavior too. Thanks!

**Comment by Luxocracy** [comment]
Apparently the Warning Glow was using a different function for determining its color. I've added the exceptions for Reaping mobs to that as well in the latest beta release.

**Comment by Nukme** [comment]
Hi, I have tested the new beta and I think it's kind of my fault to not clearly describe the request.

In the new beta, the warning glows would stay the same color as health bar. But what I actually want is that when reaping mobs fixate on a tank, there would be no warning glows instead of blue ones.

**Comment by Luxocracy** [comment]
No, the fact that they are blue isn't intended and is a flaw due to the hasty implementation of this on my part due to reusing the same code as for the healthbar.
The healthbar code returns a "safe" color for threat, while the warning glow is supposed to not return any color if it is "safe".

**Comment by Nukme** [comment]
Warning glows still exist in new release. Just return back to be constant red.

**Comment by Luxocracy** [comment]
Aaah, I forgot to test this properly from a tanks view point... This is probably why you don't do releases before going to bed.
What it does right now is return to default behaviour if the mob isn't "unsafe", which works properly for DPS, but not as a tank.

This should be working correctly for tanks now as well.

**Comment by Nukme** [comment]
Cool. Works like a charm!
